checker/build.gradle

```diff
@@ -1085,6 +1093,38 @@ task wpiPlumeLibTest(group: 'Verification') {
     }
 }
 
+// Unzip JSpecify conformance tests using the artifact
+tasks.register('unzipJSpecifyConformanceTestSuite', Copy) {
```

Comment:
I tried this different (maybe lazily evaluated) task syntax, but it didn't fix our CI problem. Feel free to undo those changes.

The offending line is [here](https://github.com/eisop/checker-framework/blob/e5bfb16b1076756288ce847f25d50df5d3a682c1/build.gradle#L393). From the documentation it sounds like that property should be settable multiple times...

Can you try minimizing what change introduces the problem? E.g. only have the tasks, but don't add the dependency on `test`, remove one of the tasks, etc. until you have a working CI.